polytiles/polytiles.py

```diff
@@ -529,88 +546,88 @@ def poly_parse(fp):
                     result = Polygon(poly)
             poly = []
             data = False
-        elif l == 'END' and not data:
+        elif l == "END" and not data:
             break
-        elif len(l) > 0 and ' ' not in l and '\t' not in l:
+        elif len(l) > 0 and " " not in l and "\t" not in l:
             data = True
-            hole = l[0] == '!'
+            hole = l[0] == "!"
         elif l and data:
             poly.append([float(x.strip()) for x in l.split()[:2]])
     return result
 
 
 def read_db(db, osm_id=0):
-    # Zero for DB bbox
+    # zero for DB BBOX
```

Comment:
BBox is an abbreviation of "Bounding Box", so it should be cased as "BBox", or "bbox" if we're fine with using it as a word.